packages/functions/src/dequantize.ts

```diff
@@ -0,0 +1,65 @@
+import { Accessor, Document, Primitive, Transform } from '@gltf-transform/core';
+import { MeshQuantization } from '@gltf-transform/extensions';
+import { createTransform } from './utils';
+
+const NAME = 'dequantize';
+
+/** Options for the {@link dequantize} function. */
+export interface DequantizeOptions {
+	/**
+	 * Pattern (regex) used to filter vertex attribute semantics for quantization.
+	 * Default: `/^(JOINTS)(_\d+)$/`.
+	 */
+	pattern?: RegExp;
+}
+
+const DEQUANTIZE_DEFAULTS: DequantizeOptions = {
+	pattern: /^(JOINTS)(_\d+)$/,
```

Comment:
Typo – this should be _excluding_ `JOINTS_*`, not including it. Presumably my manual tests worked because the CLI default is correct, but this should fail in unit tests.